src/python/pants/backend/python/goals/export.py

```diff
@@ -102,6 +102,24 @@ class ExportPluginOptions:
         advanced=True,
     )
 
+    py_hermetic_scripts = BoolOption(
+        default=True,
+        help=softwrap(
+            """
+            When exporting a mutable virtualenv for a resolve, by default
+            modify console script shebang lines to make them "hermetic".
+            The shebang of hermetic console scripts uses the python args: `-sE`:
+
+            - `-s` skips inclusion of the user site-packages directoy,
+            - `-E` ignores all `PYTHON*` env vars like `PYTHONPATH`.
+
+            Set this to false if you need non-hermetic scripts with
+            simple python shebangs that respect vars like `PYTHONPATH`.
+            """
```

Comment:
The `py_editable_in_resolve` is even more explicit about when that option applies ("This only applies when ..."), could we add that here too?

Reply:
Good point. I just pushed a commit that adds that.